modules/cloudtrail/cmdb-integration/main.tf

```diff
@@ -0,0 +1,13 @@
+module "lambda" {
```

Comment:
lets move cmdb-integration as sub-sub-module, it is mostly possible that soon we will move the cloudtrail into its separate repo so that we will have right construction of module sub-module, so that the folder `/modules/cloudtrail/cmdb-integration` goes to `/modules/cloudtrail/modules/cmdb-integration`